core/src/com/unciv/logic/civilization/Civilization.kt

```diff
@@ -191,6 +196,7 @@ class Civilization : IsPartOfGameInfoSerialization {
     var cities = listOf<City>()
     var citiesCreated = 0
     var exploredTiles = HashSet<Vector2>()
+    var exploredRegion = HashMap<String,Vector2>()
```

Comment:
Since you made this a non-transient serializable field, there should be a correspondent line in the `Civilization.clone()`